app/pages/project/classify.cjsx

```diff
@@ -230,7 +230,7 @@ module.exports = React.createClass
 
   saveClassificationAndLoadAnotherSubject: ->
     @saveClassification()
-      .then @loadAnotherSubject()
+    @loadAnotherSubject()
 
```

Comment:
This seems to work for classifications on Snapshots at Sea, and will fix the Milky Way Project.